payloads/general/USB-DriveBy/payload.txt

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Title: USB-DriveBy
+# Description: Use an USB storage device to deploy payloads on-demand
+# Author: 90N45
+# Version: 1.0
+# Category: General
+
+# Choose your preferred default network mode
+NETWORK_MODE="TRANSPARENT"
+
+LED SETUP
+
+NETMODE ${NETWORK_MODE}
+
+LED FINISH
+
+while true; do 
```

Comment:
Using this loop, the payload a appears to be repeatedly triggered. I would add a flag to make sure it only runs once.

Reply:
It's intentional... since no flag is used, you can run multiple “drive-by” payloads from the USB without having to restart the main payload on the squirrel after every single use.
The delay in [line 26](https://github.com/hak5/packetsquirrel-payloads/pull/39/files#diff-8bd0e7ddb095c1844eeeab5df7c18a811d007466835fe6a0b7f6514daf6b3866R26) provides the user enough time to unplug the USB when the “drive-by” payload finished, so that it is not unintentionally triggered again.

Reply:
Please add LED indication that the payload is running so the user knows when to unplug the USB. In its current state it would be difficult to know when the payload has been transferred and is running. I would also recommend using a variable for the payload name to make user configuration easier, just in case someone wants to use something other than `payload.txt`.